finetune/util/in_memory_finetune.py

```diff
@@ -48,10 +52,10 @@ def _evaluate(self, session):
                     k: v for k, v in self._current_finetune.saver.fallback.items() if "global_step" not in k
                 }
                 train_x, train_y = self.train_data
-                model.fit(train_x, train_y)
+                model.fit(train_x, train_y, context=self.train_context)
                 test_x, test_y = self.test_data
-                test_accuracy = np.mean(model.predict(test_x) == test_y)
-                train_accuracy = np.mean(model.predict(train_x) == train_y)
+                test_accuracy = np.mean(model.predict(test_x, context=self.test_context) == test_y)
```

Comment:
Pretty sure this metric isn't going to work

Reply:
yeah I was seeing 0s across the board. what's wrong with this metric?

Reply:
Its testing exact match of the output dictionaries